### ISSUE TITLE
Facebook Messenger

### ISSUE THREAD

**Issue by FedericoCeratto** [weechat/weechat#655]
Support to Facebook Messenger through a Python plugin would be very nice.

**Comment by flashcode** [comment]
It's possible to add native support (if the protocol is 100% free and open), but It should be already possible with Bitlbee, see: https://wiki.bitlbee.org/HowtoFacebookMQTT

**Comment by schovi** [comment]
Any progress there? I just discovered weechat and want to use it for Slack, but using it for messenger too would be best :)

**Comment by weechatter** [comment]
No. if you want slack, use the slack.py script. If you want Facebook Messenger, use bitlbee.

**Comment by Fubukimaru** [comment]
What about this plugin for slack? https://github.com/wee-slack/wee-slack

**Comment by weechatter** [comment]
This is a "script", not a "plugin", see https://weechat.org/files/doc/stable/weechat_user.en.html#plugins

**Comment by Mikaela** [comment]
If the issue is asking for support via Python plugin, doesn't that mean a Messenger script meaning https://github.com/weechat/scripts/issues rather than this issue tracker?
